sphinx_copybutton/_static/copybutton.js

```diff
@@ -76,7 +76,7 @@ const addCopyButtonToCodeCells = () => {
 
     const clipboardButton = id =>
     `<a class="copybtn o-tooltip--left" style="background-color: ${pre_bg}" data-tooltip="${messages[locale]['copy']}" data-clipboard-target="#${id}">
-      <img src="https://gitcdn.xyz/repo/choldgraf/sphinx-copybutton/master/sphinx_copybutton/_static/copy-button.svg" alt="${messages[locale]['copy_to_clipboard']}">
+      <img src="${DOCUMENTATION_OPTIONS.URL_ROOT}/_static/copy-button.svg" alt="${messages[locale]['copy_to_clipboard']}">
```

Comment:
```suggestion
      <img src="${DOCUMENTATION_OPTIONS.URL_ROOT}_static/copy-button.svg" alt="${messages[locale]['copy_to_clipboard']}">
```

This was doubling up the slash, which was breaking the images on the demo, maybe this would fix it?

Reply:
Yes, thanks. It did work for me when I checked it in the browser, I didn't realize that the generated URL had a double slash in it.